wasm/index.go

```diff
@@ -38,14 +39,43 @@ func (m *Module) populateFunctions() error {
 		return nil
 	}
 
+	// If present, extract the function names from the custom 'name' section (LLVM generates these)
```

Comment:
Even `gc` generates these. So I don't think there is a need to say "(LLVM generates these)".

btw, great work. Waiting for this to be merged :smile:

Reply:
Ahhh, sorry, got busy with other stuff.  I'll get back to this in the next few days. :roll_eyes:

Reply:
With the "LLVM generates these", no worries, can easily drop it.  Just been doing stuff pretty exclusively with LLVM generated wasm recently, so haven't checked otherwise. :wink:

Reply:
Just removed that text fragment. :smile: